### PR TITLE
Client parameters for clickhouse-test

### PR DESCRIPTION
**Pull request by KochetovNicolai**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

For changelog. Remove if this is non-significant change.

Category (leave one):
- Build/Testing/Packaging Improvement

Short description (up to few sentences):
Added `--client-option` parameter to `clickhouse-test` to pass additional parameters to client.
Several `--client-option` values are possible.

Example: `clickhouse-test --client-option experimental_use_processors=1`.

### CODE REVIEW

File: dbms/tests/clickhouse-test [hunk 1]
```diff
@@ -468,6 +508,7 @@ if __name__ == '__main__':
     group=parser.add_mutually_exclusive_group(required=False)
     group.add_argument('--shard', action='store_true', default=None, dest='shard', help='Run sharding related tests (required to clickhouse-server listen 127.0.0.2 127.0.0.3)')
     group.add_argument('--no-shard', action='store_false', default=None, dest='shard', help='Do not run shard related tests')
+    group.add_argument('--client-option', nargs='+', help='Specify additional client argument')
```
Comment (akuzm) [inline dbms/tests/clickhouse-test]: Does `nargs='+'` mean that at least one argument is always required?

Comment (KochetovNicolai) [reply dbms/tests/clickhouse-test]: No

Comment (alesapin) [reply dbms/tests/clickhouse-test]: https://docs.python.org/3/library/argparse.html#nargs
`'+'. Just like '*', all command-line args present are gathered into a list. Additionally, an error message will be generated if there wasn’t at least one command-line argument present.`

**Review by alesapin — APPROVED** [review]
LGTM

**Pull request merged**